src/components/header/header.js

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+import { Icon } from 'react-fa';
+import './header.scss';
+import Logo from '../../components/logo/logo';
+import SlackLogo from '../../images/Slack-Monochrome-White.svg';
+
+export default class Hello extends React.Component {
+  constructor() {
+    super();
+  }
+
+  render() {
+    return (
+      <div className="row-fluid">
+        <div id="header" className="col-xs-12">
+          <div className="col-xs-2 logo">
+            <Logo />
+          </div>
+
+          <div className="col-xs-7">&nbsp;</div>
+
+          <div className="social col-xs-3">
```

Comment:
Also a tiny nitpick, but semantically this is an unordered list:

```
<div className="col-xs-3">
  <ul className="social list-inline">
    <li>...</li>
    <li>...</li>
    <li>...</li>
  </ul>
</div>
```
Makes more sense, semantically.